.travis.yml

```diff
@@ -21,7 +21,9 @@ env:
     - SSL_PROXY_IMAGE="psd2-sandbox-ssl-proxy"
     - NPM_CONFIG_CACHE="$HOME/.npm"
     - OPENSHIFT_REGISTRY="openshift-registry.adorsys.de"
-    - OPENSHIFT_NAMESPACE="psd2-sandbox-dev"
+    - OPENSHIFT_NAMESPACE_ACCELERATOR="psd2-sandbox-dev"
```

Comment:
I doubt we have a danger to mix-up two products here: psd2-accelerator and xs2a-sandbox

Reply:
Should be fixed with the newest commit 👍